conftest.py

```diff
@@ -1,13 +1,11 @@
 from pyspark.sql import SparkSession
 import pytest
 
-
 # Shared across Python and docs tests
-@pytest.fixture(scope="session")
+@pytest.fixture(scope="module")
```

Comment:
Nit: Can you add a comment on this change? I assume this means any changes to `spark` will carry between tests.